README.md

```diff
@@ -1,12 +1,28 @@
-# PHPdokuwiki2findologic
+# DokuWiki Plugin: FINDOLOGIC XML Export
 
-[![Travis](https://api.travis-ci.org/TheKeymaster/phpdokuwiki2findologic.svg)](https://travis-ci.org/TheKeymaster/phpdokuwiki2findologic)
+[![Travis](https://travis-ci.org/findologic/dokuwiki-plugin-findologic-xml-export.svg?branch=master)](https://travis-ci.org/TheKeymaster/phpdokuwiki2findologic)
```

Comment:
Yeah badges! :+1:

Reply:
Note: Also edit the link to travis

Reply:
TODO:

- Change link in travis, because it actually links to the old repo.